zingolib/src/lightclient/send.rs

```diff
@@ -248,6 +250,22 @@ pub mod send_with_proposal {
 
             let step = proposal.steps().first();
 
+            // The 'UnifiedSpendingKey' we create is not a 'proper' USK, in that the
+            // transparent key it contains is not the account spending key, but the
+            // externally-scoped derivative key. The goal is to fix this, but in the
+            // interim we use this special-case logic.
+            fn usk_to_tkey(
```

Comment:
Please add a unit-test (or suite of them) demonstrating correct operation of this fn!